Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -347,6 +350,15 @@ extension ResourceFileElements {
     }
 }
 
+extension ProjectDescription.Headers {
+    fileprivate static func from(path: AbsolutePath, publicHeadersPath: String?) -> Self? {
```

Comment:
I'd add a unit test that ensures that we are setting the right value to the `public` attribute of the `Headers` instance. This will prevent future regressions in the mapping logic.

Reply:
It is tested by the `Tests/TuistDependenciesTests/SwiftPackageManager/Utils/SwiftPackageManagerGraphGeneratorTests.swift` which uses the data exposed by `Sources/TuistDependenciesTesting/DependenciesGraph/DependenciesGraph+TestData.swift`